tern/utils/general.py

```diff
@@ -45,15 +46,123 @@ def initialize_names():
 
 
 def clean_command(command):
-    '''Given a command string, clean out all whitespaces, tabs and line
-    indentations
-    Leave && alone'''
-    return re.sub(cleaning, '', command).strip()
+    '''Given a command string (only contains one command, does not contain &&
+    or ;), clean out all whitespaces, tabs and line indentations'''
+    return ' '.join(shlex.split(command))
```

Comment:
Use `shlex` to clean commands instead of `re.sub`. `shlex` can clean long whitespaces.

Reply:
> In the code check, it failed on Prospector_Linting which says:
> 
> ```
> tern/analyze/common.py
>   Line: 196
>     pep8: E741 / ambiguous variable name 'l' (col 9)
>     pep8: E741 / ambiguous variable name 'l' (col 9)
> ```
> 
> it seems that using `l` for the variable name will casue this problem.

Looks like prospector version 1.3 checks for [letters that look like numbers](https://www.flake8rules.com/rules/E741.html). I will submit a fix and rerun the build.